### PR TITLE
fix several minor typos in collections.md

### PR DESCRIPTION
**Pull request by robfallows**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/meteor/guide/pull/112%23discussion_r46448269%22%2C%20%22https%3A//github.com/meteor/guide/pull/112%23issuecomment-161378978%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/meteor/guide/pull/112%23issuecomment-161378978%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Really%20really%20appreciate%20this%21%20We%20are%20planning%20on%20doing%20a%20proof%20reading%20pass%20over%20the%20whole%20thing%20before%20an%20initial%20announcement%2C%20and%20now%20it%20will%20be%20that%20much%20easier%21%22%2C%20%22created_at%22%3A%20%222015-12-02T17%3A50%3A08Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/448783%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/stubailo%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2027ab796f88e824ef0888989a29fadc5021140305%20content/collections.md%2040%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/meteor/guide/pull/112%23discussion_r46448269%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Lol%2C%20Australian%20spelling.%22%2C%20%22created_at%22%3A%20%222015-12-02T17%3A49%3A24Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/448783%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/stubailo%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20content/collections.md%3AL150-157%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 27ab796f88e824ef0888989a29fadc5021140305 content/collections.md 40'></a> &nbsp;<b><a href='https://github.com/meteor/guide/pull/112#discussion_r46448269'>File: content/collections.md:L150-157</a></b>
- <a href='https://github.com/stubailo'></a> Lol, Australian spelling.
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/meteor/guide/pull/112#issuecomment-161378978'>General Comment</a></b>
- <a href='https://github.com/stubailo'></a> Really really appreciate this! We are planning on doing a proof reading pass over the whole thing before an initial announcement, and now it will be that much easier!

<a href='https://www.codereviewhub.com/meteor/guide/pull/112?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/meteor/guide/pull/112?approve=1'></a>&nbsp;<a href='https://github.com/meteor/guide/pull/112'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by stubailo** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/meteor/guide/pull/112'>Refresh</a>.<a href='#crh-update-none'></a>

File: content/collections.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ Lists.schema = new SimpleSchema({
 });
 ```
 
-The issue with this schema is that due to the DDP behaviour just mentioned, each change to *any* todo item in a list will require sending the *entire* set of todos for that list over the wire. This is because DDP has no concept of "change the `text` field of the 3rd item in the field called `todos`", simply "change the field called `todos` to (say) `[{text: 'first'}, {text: 'second'}]`".
+The issue with this schema is that due to the DDP behavior just mentioned, each change to *any* todo item in a list will require sending the *entire* set of todos for that list over the wire. This is because DDP has no concept of "change the `text` field of the 3rd item in the field called `todos`", simply "change the field called `todos` to (say) `[{text: 'first'}, {text: 'second'}]`".
```
Comment (stubailo) [inline content/collections.md]: Lol, Australian spelling.

**Comment by stubailo** [comment]
Really really appreciate this! We are planning on doing a proof reading pass over the whole thing before an initial announcement, and now it will be that much easier!

File: content/collections.md (hunk 1 above)
Comment (tmeasday) [inline content/collections.md]: *shudder*

Comment (tmeasday) [inline content/collections.md]: To be fair, it's English spelling ;) But I agree we should use US spelling.

Comment (robfallows) [inline content/collections.md]: Being from the UK myself, I tend to shudder along with Tom. But it's all for the greater good :wink:

**Comment by robfallows** [comment]
You're most welcome. BTW I have found a couple of places with some rather "clumsy" wording, making the narrative difficult to follow. PR?

**Comment by stubailo** [comment]
We'd definitely appreciate it! But keep in mind that we will probably do an extra pass over the wording anyway in the next step of development. Even more useful than fixing wording would be making sure the content is correct and we didn't miss anything important.

**Comment by robfallows** [comment]
> Even more useful than fixing wording would be making sure the content is correct and we didn't miss anything important.

So, I'm looking right at the end of the doc - the mdg:cursor-utils package. I can't find this anywhere - does it exist? Also (without knowing anything about it) I assume it requires a cursor to work from, yet the example code seems to start with a `findOne` - which returns an array.

**Comment by tmeasday** [comment]
Ahh yeah. The cursor-utils package. @stubailo I forgot about this, let's discuss

**Pull request merged**